lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -1755,6 +1755,85 @@ setReadOnly( Complex64Array.prototype, 'some', function some( predicate, thisArg
 	return false;
 });
 
+/**
+* Returns subarray elements.
+*
+* @name subarray
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {integer} [begin] - starting index (inclusive)
+* @param {integer} [end] - ending index (non-inclusive)
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} arguments must be integers
```

Comment:
```suggestion
* @throws {TypeError} first argument must be an integer
* @throws {TypeError} second argument must be an integer
```

We typically make explicit the errors for each argument and don't combine into a single annotation.